src/devices/Ssd13xx/samples/Program.cs

```diff
@@ -206,12 +207,16 @@ void DisplayClock(Ssd1306 ssd1306)
     {
         using (Image<Rgba32> image = new Image<Rgba32>(128, 32))
         {
+            if (image.TryGetSinglePixelSpan(out Span<Rgba32> imageSpan))
```

Comment:
You can use `Clear(Color.Black)` here to flood fill the image without any color blending.